src/prefect/_internal/compatibility/blocks.py

```diff
@@ -0,0 +1,27 @@
+import inspect
+from typing import Any, Union
+
+from prefect.filesystems import NullFileSystem, WritableFileSystem
+
+
+async def _call_explicitly_async_block_method(  # pyright: ignore[reportUnusedFunction]
```

Comment:
Since this is in `prefect._internal`, we could remove the leading `_` to appease `pyright`.
```suggestion
async def call_explicitly_async_block_method(
```